boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_common.dts

```diff
@@ -190,11 +189,11 @@
 			/* Secure image memory */
 		};
 
-		sram0_bsd: image_bsd@20010000 {
+		sram0_bsd: image_bsd@20016000 {
```

Comment:
@anangl  I am not sure this will work out-of-the-box
IIRC, the sram0_bsd needs to be on this specific location.
I wonder if we could leave this untouched, or create a sample overlay for the partitions (just to have the samples working properly)